README.md

```diff
@@ -1,52 +1,192 @@
-## BEGIN - TO DELETE TILL END
+# reshuffle-shopify-connector
 
-THIS IS A TEMPLATE REPO FOR NEW RESHUFFLE CONNECTORS
-1. Create a new connector repo from this template using this link https://github.com/reshufflehq/reshuffle-template-connector/generate
-2. Clone the repo locally
-3. Rename all occurrences of _CONNECTOR_NAME_
-4. `npm install`
-5. `npm run build:watch`
-6. Implement your events/actions in `src/index.ts`
-7. `npm run lint`
-8. Push your code
-9. Go to https://app.circleci.com/projects/project-dashboard/github/reshufflehq/
-    a. You should see your new connector repo
-    b. click on `Set Up Project` for the repo
-    c. click on `Use Existing Config`
-    d. click on `Start Building`
+[Code](https://github.com/reshufflehq/reshuffle-shopify-connector) |  [npm](https://www.npmjs.com/package/reshuffle-shopify-connector) | [Code sample](https://github.com/reshufflehq/reshuffle/tree/master/examples/shopify)
 
-10. If circle CI checks are all green, you are all set!
+`npm install reshuffle-shopify-connector`
 
-// Keep documentation template below
+This connector uses [Official Node Shopify Client](https://www.npmjs.com/package/shopify-api-node) package.
 
-## END
+### Reshuffle Shopify Connector
 
-# reshuffle-_CONNECTOR_NAME_-connector
+This connector provides a connector for [Shopify](https://www.shopify.com/online).
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+The following example listen to Shopify events:
```

Comment:
should be `listens`. Also - this listens to a specific event, no? so maybe we should say `listens to Shopify fulfillment events` ?